roles/kubernetes/node/tasks/install.yml

```diff
@@ -21,7 +21,7 @@
 
 - name: install | Copy kubelet binary from download dir
   synchronize:
-    src: "{{ local_release_dir }}/hyperkube"
+    src: "{{ local_release_dir }}/kubelet"
```

Comment:
you probably want to use the download.kubelet.dest  instead

Reply:
Do you want to mix vars that way between roles?